install_config/install/advanced_install.adoc

```diff
@@ -60,6 +60,11 @@ xref:../../install_config/install/rpm_vs_containerized.adoc#install-config-insta
 Containerized Hosts] to ensure that you understand the differences between these
 methods, then return to this topic to continue.
 
+If you are interested in optimizing your installation, including recommendations
```

Comment:
@jeremyeder Sure. Changed to:

"For large-scale installs, including suggestions for optimizing install time,
see the Scaling and Performance Guide."

Let me know if there's anything else.